comp/core/tagger/collectors/workloadmeta_main.go

```diff
@@ -25,6 +27,7 @@ const (
 	workloadmetaCollectorName = "workloadmeta"
 
 	staticSource         = workloadmetaCollectorName + "-static"
+	hostSource           = workloadmetaCollectorName + "-host"
```

Comment:
staticSource is already covering some host tags that are set with the `DD_TAGS` envvar or `tags` config option.

But here I suppose we also want to get tags from the host provider?

Reply:
The host tags supplied with static source are not short-lived so I think it makes sense to have a new `hostSource` Ive updated this in my last commit - with a host source provided by a host tags collector.